src/nginx-check

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+source /etc/envvars
+NGINX_TEST=$(nginx -t 2>&1)
+
+echo $NGINX_TEST > /etc/nginx/status.txt
+
+if [[ $NGINX_TEST == *"failed"* ]]
+then
+  echo "ngicon.conf.[app=$APP]ok:2|g" > /dev/udp/localhost/8125
+elif [[ $NGINX_TEST == *"success"* ]]
+then
+  echo "ngicon.conf.[app=$APP]ok:0|g" > /dev/udp/localhost/8125
```

Comment:
Would it work if you took out the `sv hup nginx` from `consul-template.service` and stuck it here once?  We wouldn't want to reload the config if we don't see `success` anyway.

Reply:
I was trying to keep the functionalities separate. Theres a cron job that will run the check every minute, I didn't think it'd be a good idea to `hup` every minute if things don't need to.

Reply:
Ah I see - This same check is being used in the cron job 👍